scripts/update

```diff
@@ -29,41 +29,44 @@ function run_database_migrations() {
     else
         # Initialize the database for migrations.
         docker-compose \
-            run --rm api-server \
+            run --rm sbt \
             "migrations/run init"
     fi
 
     # Run database migrations. The way scala-forklift works requires this to be called twice:
     # the first run figures out the migrations to run and the second run applies them.
     docker-compose \
-        run --rm api-server \
+        run --rm sbt \
         ";migrations/run update;migrations/run apply"
 }
 
-if [ "${BASH_SOURCE[0]}" = "${0}" ]
-then
-    if [ "${1:-}" = "--help" ]
-    then
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
         usage
     else
         echo "Updating Scala dependencies"
-        docker-compose up -d api-server
+        docker-compose \
+            run --rm --no-deps sbt \
+            update
 
-        docker-compose exec -T api-server ./sbt update
-        docker-compose rm -sf api-server
+        echo "Building Scala assembly JARs"
+        docker-compose \
+            run --rm --no-deps sbt \
+            ";api/assembly;backsplash-server/assembly"
```

Comment:
Since we're running the assembly JARs in `server`, they should be created in `update` as part of bootstrapping the development environment, etc.